src/views/Routes.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { Router, Route, browserHistory } from 'react-router'
+// import { Provider } from 'react-redux'
+
+
+import AppLayout from './AppLayout'
+import { HomeRoute } from './home'
+import { ObjectRoute } from './objects/object'
+
+const Routes = () => {
+  return (
+    <Router history={browserHistory}>
+      <Route component={AppLayout}>
+        <Route path="/" component={HomeRoute} />
+        <Route path="/object" component={ObjectRoute} />
+      </Route>
+    </Router>
+  )
+}
+
+// const Routes = (props) => {
```

Comment:
This is commented out because we are not using redux, I left it in as reference for when we integrate redux